### PR TITLE
small test app for verifying instrumentation

### PR DESCRIPTION
**Pull request by kubawach**
**Description:**

Simple test app for verifying that an instrumentation has been applied. Prints out "SUCCESS" if a http request contains traceparent header, FAILURE otherwise.
The app is used for static instrumentation plugin integration test.

### CODE REVIEW

**Review by Enkelian — APPROVED** [review]
Thanks!

File: static-instrumenter/test-app/build.gradle.kts [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+tasks {
+  withType<JavaCompile>().configureEach {
+    with(options) {
+      release.set(11)
+    }
+  }
+
+  jar {
+    manifest {
+      attributes(mapOf("Main-Class" to "io.opentelemetry.contrib.staticinstrumenter.test.HttpClientTest"))
```
Comment (trask) [inline static-instrumenter/test-app/build.gradle.kts:23]:
```suggestion
      attributes("Main-Class" to "io.opentelemetry.contrib.staticinstrumenter.test.HttpClientTest")
```

**Review by trask — APPROVED** [review]

File: static-instrumenter/test-app/src/main/java/io/opentelemetry/contrib/staticinstrumenter/test/HttpClientTest.java [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+
+  private HttpClientTest() {}
+
+  public static void main(String[] args) throws Exception {
+    System.setProperty("otel.traces.exporter", "logging");
+    makeCall();
+  }
+
+  private static void makeCall() throws IOException {
+    try (CloseableHttpClient httpClient = HttpClients.createDefault()) {
```
Comment (anuraaga) [inline static-instrumenter/test-app/src/main/java/io/opentelemetry/contrib/staticinstrumenter/test/HttpClientTest.java:27]: Not a big deal but would recommend using OkHttp instead if possible, it's our general "go-to" HTTP client, hence already being in dependency management

File: static-instrumenter/test-app/src/main/java/io/opentelemetry/contrib/staticinstrumenter/test/HttpClientTest.java [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+      HttpGet request = new HttpGet("https://httpbin.org/get");
+      try (CloseableHttpResponse response = httpClient.execute(request)) {
+        HttpEntity entity = response.getEntity();
+        System.out.println();
```
Comment (anuraaga) [inline static-instrumenter/test-app/src/main/java/io/opentelemetry/contrib/staticinstrumenter/test/HttpClientTest.java:31]: Needed?

Comment (kubawach) [reply static-instrumenter/test-app/src/main/java/io/opentelemetry/contrib/staticinstrumenter/test/HttpClientTest.java:31]: Not at all - thanks for pointing this out.

File: static-instrumenter/test-app/src/main/java/io/opentelemetry/contrib/staticinstrumenter/test/HttpClientTest.java (hunk 2 above)
Comment (kubawach) [reply static-instrumenter/test-app/src/main/java/io/opentelemetry/contrib/staticinstrumenter/test/HttpClientTest.java:27]: Oh, next time I'll default to OkHttp then ;)

**Review by mateuszrzeszutek — APPROVED** [review]

**Pull request merged**